src/Schema/Events/Click.ts

```diff
@@ -1607,3 +1607,32 @@ export interface ClickedOnDuplicateArtwork {
   label: string
   flow: string
 }
+/**
+ *  User clicks on one of the buttons on the validation address modal.
+ *
+ *  This schema describes events sent to Segment from [[clickedValidationAddressOptions]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "clickedValidationAddressOptions",
+ *    context_module: "OrdersShipping",
+ *    context_page_owner_type: "orders-shipping",
+ *    context_page_owner_id: "57e60c68-a198-431e-8a02-6ecb01e3a99b",
+ *    user_id: "61bcda16515b038ce5000104"
```

Comment:
Hey, if this is going to be the logged-in user `user_id`, you probably don't need to set it since it's usually added by default to analytics events.